src/constants.ts

```diff
@@ -37,6 +37,7 @@ export const nativeMainnetChainIdentifiers: string[] = [
   "neutron",
   "gitopia",
   "likecoin-mainnet",
+  "doravota",
```

Comment:
@tomas-dora Could you remove this values? It is for Keplr Infra's nodes.

Reply:
@HeesungB  Is the configuration here for here? (https://wallet.keplr.app/chains)

What should I do if I want to join here?

Reply:
I have deleted it, you can check if it can fulfill the conditions of merge. Regarding how to add our chain here, we can add it later